tests/integ/test_packaging.py

```diff
@@ -497,33 +534,6 @@ def test_add_requirements_with_ranged_requirements_in_yaml(session, ranged_yaml_
         session.add_requirements(ranged_yaml_file)
 
 
```

Comment:
Packages can no longer be added directly during sproc/udf registration because of the new parameter.

Reply:
Using how we enable/disable sql simplifier might be useful again here. Make this part of session which is accessible during udf/sproc creation.